src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1963,7 +1993,7 @@ namespace UnitarySystems {
             state.dataSize->DXCoolCap = EqSizing.DesHeatingLoad;
         }
 
-        if (this->m_OKToPrintSizing && this->UnitType != "CoilSystem:Cooling:DX") PrintFlag = true;
+        if (this->m_OKToPrintSizing && !this->m_IsCoilSystemCoolingDX) PrintFlag = true;
```

Comment:
This should be changed to && this->m_IsUnitarySystem

Reply:
👍  on changing a string comparison to an integer comparison!

Reply:
This is not a giant deal, but I notice that in this module we are using `this->m_X` convention (brings me back to my Microsoft days). I am not against using either `this->` or `m_` necessarily, but I just want to point out that together they are somewhat redundant. All member variables implicitly have a `this->` in front of them. Yes, there is implicit precedence of local/parameter, member, and global variables with the same names and the `this->` can be used to override local/member precedence. But by using `m_` for member variables you are essentially saying that member variables have different names than local variables, so the need for explicit `this->` will no longer come up. You are also "self-documenting" the variable to be a member variable and so `this->` and `m_` are both serving as annotations that this is a member variable. One is probably enough.